packages/core/test/__snapshots__/Select.js.snap

```diff
@@ -42,6 +40,7 @@ exports[`Select renders 1`] = `
   border-width: 1px;
   border-style: solid;
   border-color: #c0cad5;
+  border-color: #007aff;
```

Comment:
double `border-color` with different values?

Reply:
Addressed in https://github.com/priceline/design-system/pull/683